ChangeLog

```diff
@@ -6,6 +6,11 @@ What's New in astroid 2.13.0?
 =============================
 Release date: TBA
 
+* Fixed importing of modules that have the same name as the file that is importing.
+  A file called ``math.py`` will now correctly ``import math`` from the standard library
+  instead of trying to import itself.
```

Comment:
Up to you, if you like this better.
```suggestion
  ``astroid`` will now correctly view a file called ``math.py`` as importing math from the
    standard library when it calls ``import math`` instead of treating it as importing itself.
```

Reply:
What about:
```suggestion
  ``astroid`` will now correctly handle an ``import math`` statement in a file called ``math.py``
  by relying on the import system.
```

Reply:
I like it.